Assets/scripts/Inputs/InputIJKLManager.cs

```diff
@@ -1,34 +1,44 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
+using Interfaces.ServiecesInterface;
+using Replay_Service;
 using UnityEngine;
 
 public class InputIJKLManager : MonoBehaviour
 {
     Controls controls = Controls.IJKL;
+    InputData inputData;
+    private void Start()
+    {
+        ServiceLocator.Instance.get<IInputManager>().AddPlayerInputData(controls);
```

Comment:
either use Manager or Service - dont mix both names in a single project

Reply:
okay